pnpm-lock.yaml

```diff
@@ -1,4 +1,4 @@
-lockfileVersion: '6.0'
+lockfileVersion: '6.1'
```

Comment:
do you know why we downgraded to 6.0 again? I always have to remove this from my PR too